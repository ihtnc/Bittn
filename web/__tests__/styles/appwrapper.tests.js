import React from 'react';
import renderer from 'react-test-renderer';

import AppWrapper from "@styles/appwrapper";

const shallow = (comp) => renderer.create(comp).toJSON();

describe("AppWrapper styled component", () => {
  test("should match latest snapshot", () => {
    const wrapper = shallow(<AppWrapper />)
    expect(wrapper).toMatchSnapshot();
  });
});