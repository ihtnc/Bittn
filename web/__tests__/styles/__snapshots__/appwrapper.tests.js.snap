// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppWrapper styled component should match latest snapshot 1`] = `
<section
  className="css-xjel9q-AppWrapper e1q1sy7m0"
/>
`;
